src/layout/Map/config.ts

```diff
@@ -34,6 +34,101 @@ export const Config = new CG.component({
+            new CG.str()
+              .optional()
+              .setTitle('Attribution')
+              .setDescription('Ascribing a work or remark to a particular unit for recognition'),
+          ),
+          new CG.prop(
+            'subdomains',
+            new CG.arr(new CG.str())
+              .optional()
+              .setTitle('Subdomains')
+              .setDescription(
+                'List of subdomains. Used for balancing the load on different map tiling servers. ' +
+                  'A random one will replace {s} in the defined url.',
+              ),
+          ),
+          new CG.prop(
+            'format',
+            new CG.str()
+              .optional({ default: 'image/jpeg' })
+              .setTitle('Format')
+              .setDescription(
+                'The MIME type of the WMS tiles to request from the WMS server, as specified by the WMS standard',
+              ),
+          ),
+          new CG.prop(
+            'version',
+            new CG.str()
+              .optional({ default: '1.1.1' })
+              .setTitle('WMS Version')
+              .setDescription('The version of the WMS standard to use'),
+          ),
+          new CG.prop(
+            'opacity',
+            new CG.num()
+              .optional({ default: 1.0 })
+              .setTitle('Layer Opacity')
+              .setDescription('The opacity of the WMS layer, ranging from 0 (fully transparent) to 1 (fully opaque)'),
+          ),
+          new CG.prop(
+            'zoomOffset',
+            new CG.num()
+              .optional({ default: 0 })
+              .setTitle('Zoom Offset')
+              .setDescription('The zoom offset for the WMS layer'),
+          ),
+          new CG.prop(
+            'tileSize',
+            new CG.num()
+              .optional({ default: 2048 })
+              .setTitle('Tile Size')
+              .setDescription('The size of the tiles in pixels'),
+          ),
+          new CG.prop(
+            'transparent',
+            new CG.bool()
+              .optional({ default: false })
+              .setTitle('Transparency')
+              .setDescription('Whether the WMS layer should be transparent'),
+          ),
+          new CG.prop(
+            'uppercase',
+            new CG.bool()
+              .optional({ default: false })
+              .setTitle('Uppercase')
+              .setDescription('Whether the WMS parameters should be uppercase'),
+          ),
+          new CG.prop(
+            'minZoom',
+            new CG.num()
+              .optional({ default: 0 })
+              .setTitle('Minimum Zoom Level')
+              .setDescription('The minimum zoom level for the WMS layer'),
+          ),
+          new CG.prop(
+            'maxZoom',
+            new CG.num()
+              .optional({ default: 18 })
+              .setTitle('Maximum Zoom Level')
+              .setDescription('The maximum zoom level for the WMS layer'),
+          ),
+        )
```

Comment:
As far as I can see, `opacity`, `zoomOffset`, `tileSize`, `minZoom` and `maxZoom` are standard props that also apply to standard layers that we have not exposed in the config for the standard `layers`-prop. Is there a specific reason these are needed for WMS-layers?

Reply:
I included `opacity`, `zoomOffset` and `tileSize`, since we have used them for our own wms setup, but i am now reliazing that we use them with the default values. There should not be a need for them so i will remove them.

When it comes to `minZoom` and `maxZoom`, WMS layers dont have spesific needs for it that normal tileLayers do not. The reason they are included is because of a need to hide a certain tile-/wms-layer until the user has zoomed in a spesific amount. The WMS layer that DMF uses is one that contains alot of gemoetries, and that can cause a lot of clutter and confusion when zoomed out beyond a certain point. Therefore we use `minZoom` and `maxZoom` to hide the layer until needed.

Reply:
Ait, thats fair, you can keep them 👍  I guess it should not really hurt as we are already completely bound by leaflet's API already